core/App/ehsm_core_test.cpp

```diff
@@ -54,44 +54,49 @@ step3. decrypt the cipher text by CMK correctly
 void test_AES128()
 {
     char* returnJsonChar = nullptr;
-    char* plaintext = "Test1234-AES128";
-    char* aad = "challenge";
+    char* plaintext = StringToChar("Test1234-AES128");
```

Comment:
curious about how the compiling passes? because it's defined as "static" in the other file:

static char* StringToChar(string str)

Reply:
What's the difference:
1. char* plaintext = StringToChar("Test1234-AES128");
2. char* plaintext = "Test1234-AES128";
Can use  char plaintext[] = "Test1234-AES128" ?